MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+		var extent = (double)container.GetType().GetRuntimeProperties().FirstOrDefault(x => x.Name == "TotalExtent").GetValue(container);
+		listView.HeightRequest = extent;
+	}
+}
+{% endhighlight %}
+{% endtabs %}
+
+
+When `SfListView` is in [AutoFitMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_AutoFitMode) as `Height`, the extend of the ListView will be updated only while scrolling. So you can resize the ListView in [VisualContainer]((https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.VisualContainer.html)) `PropertyChanged` method as like below.
+
+{% tabs %}
+{% highlight xaml %}
+<ScrollView>
+   <sync:SfListView x:Name="listView" AutoFitMode="Height" ItemsSource="{Binding BookInfo}" Loaded="listView_Loaded"/>
+</ScrollView>
+{% endhighlight %}
+{% highlight C# %}
+using Syncfusion.ListView.Control.Helpers;
+public partial class MainPage : ContentPage
+{
+    VisualContainer visualContainer;
+    bool loaded;
+
+    public MainPage()
+    {
+        InitializeComponent();
+        visualContainer = listView.GetVisualContainer();
+        visualContainer.PropertyChanged += VisualContainer_PropertyChanged;
+    }
+
+    private void VisualContainer_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+    {
+        if (e.PropertyName == "Height" && listView.HeightRequest != visualContainer.Height && loaded)
+            listView.HeightRequest = visualContainer.Height;
+    }
+
+    private void listView_Loaded(object sender, ListViewLoadedEventArgs e)
+    {
+        var extent = (double)visualContainer.GetType().GetRuntimeProperties().FirstOrDefault(x => x.Name == "TotalExtent").GetValue(visualContainer);
+        listView.HeightRequest = extent;
+        loaded = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+
+N> While loading in `AutoFitMode` make sure that the [ItemSize](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemSize) property value is not specified, to avoid extra space below the list. Update the size of the container after `ListView` loaded to render all the list items in the view.
+
+The following limitations should be noted when using the previous approaches:
+
+* As the entire list items are loaded inside the parent `ScrollView` element, the [ItemAppearing](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemAppearing) and [ItemDisappearing](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemDisappearing) events will not be fired when scrolling.
+* When performing keyboard navigation, the view cannot be scrolled automatically while navigating out of view.
+* Scrolling through the touch action will be handled in all platforms and `SfListView` scrolling will be handled by the parent `ScrollView`.

```

Comment:
Touch action scrolling will be handled in all platforms and `SfListView` scrolling will be handled by the parent `ScrollView`.

Reply:
I have modified the line as per the concerns provided